curriculum/challenges/english/25-front-end-development/workshop-merge-sort/656680b0fc79f2c38a34d90e.md

```diff
@@ -0,0 +1,84 @@
+---
+id: 656680b0fc79f2c38a34d90e
+title: Step 8
+challengeType: 20
+dashedName: step-8
+---
+
+# --description--
+
+Now it's time to sort and merge the lists (`left_part` and `right_part`) into the original list. 
+
+You can do this by comparing elements on both lists, and merging the smaller element to the main list. You are going to do this comparison for all the indexes in `left_part` and `right_part`.
+
+Create three variables: `left_array_index`, `right_array_index`, and `sorted_index` and set their values to `0`. These variables will help you keep track of each index during the sorting process.
```

Comment:
out of curiosity, do we teach chained assignment? should we? is that something commonly used in Python?

it looks like the tests are expecting individual lines for each variable and assignment. 
but should we accept this answer?

```py
    left_array_index = right_array_index = sorted_index = 0
```

Reply:
So should we go with this:

```
left_array_index = 0 
right_array_index = 0 
sorted_index = 0
```